test_regress/t/t_initialstatic_circ.pl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2022 by Antmicro Ltd. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+compile(
+    verilator_flags2 => ["--trace"]
```

Comment:
Do we need the `--trace` for this test to cover what it intends to cover?

Reply:
Without ``--trace`` option, verilator is optimizing away ``id`` node from the test.

I've changed the test a bit and now, ``--trace`` option is not required.